src/AppBundle/Search/SearchResultsProvider.php

```diff
@@ -40,12 +39,32 @@ public function find(SearchParametersFilter $search): array
         ));
     }
 
-    public function findCommittees(): array
+    public function findCommittees(SearchParametersFilter $search): array
     {
-        // the following part is a fixture only, for this issue
+        if ($coordinates = $search->getCityCoordinates()) {
+            $qb = $this->committee
+                ->createNearbyQueryBuilder($coordinates)
+                ->orderBy('distance_between', 'asc')
+                ->andWhere($this->committee->getNearbyExpression().' < :distance_max')
+                ->setParameter('distance_max', $search->getRadius())
+            ;
+        } else {
+            $qb = $this->committee->createQueryBuilder('c');
+        }
+
+        if (!empty($search->getQuery())) {
```

Comment:
`if ($searchQuery = $search->getQuery())) {`, then reuse the var two lines below.